Src/WitsmlExplorer.Frontend/components/StyledComponents/Button.tsx

```diff
@@ -47,13 +67,42 @@ const GhostButton = styled(EdsButton)<{ colors: Colors }>`
 
 const GhostIconButton = styled(EdsButton)<{ colors: Colors }>`
   white-space: nowrap;
-  color: ${(props) =>
-    !props.color || props.color === "primary"
-      ? props.colors.infographic.primaryMossGreen
-      : undefined};
+  color: ${(props) => props.colors.infographic.primaryMossGreen};
+`;
+
+const TableIconButton = styled(EdsButton)<{
+  colors: Colors;
+  userTheme: UserTheme;
+}>`
+  white-space: nowrap;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -55%);
+  ${(props) =>
+    props.userTheme === UserTheme.Compact &&
+    css`
+      height: 22px;
+      width: 22px;
+      &::after {
+        width: 22px;
+        height: 22px;
+      }
+    `}
+  ${(props) =>
+    (!props.color || props.color === "primary") &&
+    css<{ colors: Colors }>`
+      color: ${(props) => props.colors.infographic.primaryMossGreen};
+    `}
 `;
 
 const OutlinedButton = styled(EdsButton)<{ colors: Colors }>`
   white-space: nowrap;
   color: ${(props) => props.colors.infographic.primaryMossGreen};
 `;
+
+export const TableIconButtonLayout = styled.div`
```

Comment:
Is this layout something we want to export?

Reply:
Removed